### ISSUE TITLE
Enhancement: a Possibility to do it on a repeat time

### ISSUE THREAD

**Issue by DavidRoseLincs** [DJDavid98/HammerTime#25]
A Possible Idea No idea how limited you might be to do it BUT
maybe having an option where it repeats weekly FOR example for streamers (not myself)

This Streamers stream starts on (STREAM DAY: TIME : OPTIONAL Daily-Weekly-Monthly-Yearly)

so everyone week it will say that date so (27th this week 4th next week) Both sharing the same time, of course, I might be asking the wrong person might be better to ask the person who codes the conversion not the person who does the website but asking is better than not at all
(I would do this myself but I don't know the language D:)

Sorry if I made any spelling mistakes and I am not explaining it properly will happily explain myself if I am confusing not the best at English.

**Comment by DavidRoseLincs** [comment]
cant remember how to add the Enhancement label D:

**Comment by DJDavid98** [comment]
This feels to me like it's completely outside the scope of this project. You can provide some examples on what exactly you expect to see in the URL and what the app should do with that, but I'm really leaning towards not adding this kind of functionality.

**Comment by DavidRoseLincs** [comment]
To be fair, I gave it a lot more thinking and came to the conclusion that what I meant made no sense. I apologise for wasting your time.

The original concept was to save time by not having to update the message every week for the same day and to give those who use the same time every week on the same day a week later a better quality of life when it came to saving them time editing it every week with the same time but different day.

I realized it didn't make much sense why I was going to close the issue when I got home, especially because it wasn't something I believed was possible to do without obviously adding another formatting option to the ones that already existed.
